integreat_cms/release_notes/current/unreleased/2706.yml

```
@@ -0,0 +1,2 @@
+en: Add standardized timestamp for all dates like last update, created at, last usage, last login, date added
+de: Füge einen standardisierten Zeitstempel für alle Daten, wie letzte Aktualisierung, erstellt am, letzte Nutzung, letzte Anmeldung, Datum des Hinzufügens hinzu
```

I suggest using only nouns, as this contributes to a more formal and precise expression.

```suggestion
de: Füge einen standardisierten Zeitstempel für alle Daten, wie letzte Aktualisierung, Erstellungsdatum, letzte Nutzung, letzte Anmeldung, Datum des Hinzufügens hinzu
```